test/CMakeLists.txt

```diff
@@ -18,7 +18,11 @@ FetchContent_MakeAvailable(googletest)
 include(GoogleTest)
 include(CTest)
 
-add_compile_options(-W -Wall -Wextra)
+# TODO this should be added only for our code, it is making library code fail
```

Comment:
Check this again, there were actual errors inside esp-dsp which are now fixed

Reply:
Lots of Wsign-compare Wformat from esp-dsp. Need to add the warning flags to Shrapnel code only.